### ISSUE TITLE
Racket 8.0: `raco test` broken

### ISSUE THREAD

**Issue by yurkobb** [NixOS/nixpkgs#116086]
**Describe the bug**
Running `raco test` fails when running with more than one file on the command line. Possibly related to #114993. 

**To Reproduce**
1. Create a `temp.rkt` file with the following contents:
    ```racket
    #lang racket/base
    
    (require rackunit)
    
    (check-equal? 0 42)
     ```
2. `cp temp.rkt temp2.rkt`
3. run `raco test temp.rkt temp2.rkt`
4. get an error:
    ```
    raco test: "temp.rkt"
    temp.rkt: find-exe: can't find Racket executable for variant cs
    raco test: "temp2.rkt"
    temp2.rkt: find-exe: can't find Racket executable for variant cs
    2/2 test failures
    ```

**Notify maintainers**
@henrytill @vrthra

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.18, NixOS, 20.09.3403.1c7f02b0b9f (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.10`
 - channels(root): `"nixos-20.09.3436.6be212cce73, unstable-21.05pre275822.916ee862e87"`
 - channels(yurb): `"nixos-20.03-20.03.2351.f8248ab6d9e"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: racket
```

**Comment by yurkobb** [comment]
Probably fixed by #114993

**Comment by RaitoBezarius** [comment]
Cannot reproduce with instructions, closing, reopen if it is indeed still broken.

```
/tmp via ❄️  impure (shell) 
❯ raco test temp.rkt temp2.rkt
raco test: "temp.rkt"
--------------------
FAILURE
name:       check-equal?
location:   temp.rkt:5:0
actual:     0
expected:   42
--------------------
raco test: "temp2.rkt"
--------------------
FAILURE
name:       check-equal?
location:   temp2.rkt:5:0
actual:     0
expected:   42
--------------------
2/2 test failures
```
